src/bin/firebase.js

```diff
@@ -18,7 +18,19 @@ if (!semver.satisfies(nodeVersion, pkg.engines.node)) {
 }
 
 const updateNotifier = require("update-notifier")({ pkg: pkg });
-updateNotifier.notify({ defer: true, isGlobal: true });
+const clc = require("cli-color");
+const TerminalRenderer = require("marked-terminal");
+const marked = require("marked");
+marked.setOptions({
+  renderer: new TerminalRenderer(),
+});
+const updateMessage =
+  `Update available ${clc.xterm(240)("{currentVersion}")} → ${clc.green("{latestVersion}")}\n` +
```

Comment:
I tried to copy the style from update-notifier's default message as closely as possible. Unfortunately, they use `chalk` instead of `cli-color` for styling, and there isn't an equivalent method in `cli-color to `chalk.cim()`. `xterm(240)` is pretty close, however.